media/transcoder/src/test/java/com/example/transcoder/CreateJobFromAdHocTest.java

```diff
@@ -120,12 +120,23 @@ public void test_CreateJobFromAdHoc() throws Exception {
     assertThat(output, containsString(jobName));
     String[] arr = output.split("/");
     JOB_ID = arr[arr.length - 1].replace("\n", "");
-    bout.reset();
 
-    Thread.sleep(300000);
+    int attempts = 5;
+    boolean succeeded = false;
+
+    while (attempts > 0 && !succeeded) {
+      Thread.sleep(60000);
```

Comment:
[Please use the `java.time` package here instead](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/master/SAMPLE_FORMAT.md#time):
```suggestion
      TimeUnit.MINUTES.sleep(1);
```